### ISSUE TITLE
dcm command line distribution broken?

### ISSUE THREAD

**Issue by samdjones** [ibm-datapower/datapower-configuration-manager#26]
It appears the command line distribution is broken, I see this error when I try to use DCM (any target):

`BUILD FAILED:
[DCM root dir]/dcm-distros/dcm.1/deploy.ant.xml:137: The Ant property 'dcm.dir' doesn't seem to point to DCM since [DCM root dir]/dcm-distros/dcm.1/dist/dcm.dev.jar is not found.`

I believe this is due to build.xml line 35:

`<property name="version" value="dev"/>  <!-- default -->`

Before the value was "1", which produces a jar named "dcm.1.jar", which deploy.ant.xml expects. Reverting the value to "1" fixes the command line distribution easily enough, but I don't know if that breaks the UCD plugin distribution. I would try but I don't have a UCD setup right now. Maybe later this week.

IMO, having version numbers in _internal_ build artefacts like this is a bad idea and just causes build maintenance problem (like this). It is OK putting a version number in the _external_ distribution .zip file (e.g. DCM.1.0.zip and DataPowerUCDPlugin.1.0.zip) but I suggest keeping file names _inside_ those packages consistent.

I see a task in build.xml (line 99):

`<replace file="tmp/dcm/deploy.ant.xml" token="value=&quot;1&quot;" value="value=&quot;${version}&quot;"/>`

This seems to patch the DCM command line distribution to work with UCD. Why does this all need to happen? Have I missed something?

Cheers,
Sam

**Comment by richgroot** [comment]
Sam,

I agree that the versioning in the filename is both unnecessary and a
source of trouble.  If you propose a pull request to get rid of it I could
predict with some confidence that it would be quickly accepted.  Changing
the default naming of the files might break existing scripts that people
may have, but we already crossed that bridge when we renamed the plugin jar
file, so I'd just keep going at this point.

Thanks,
Rich
